### ISSUE TITLE
[BUG] No sound

### ISSUE THREAD

**Issue by GodAtum360** [linuxserver/docker-freetube#3]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Installed Freetube using Docker Compose. Everything works fine apart from there's no sound. 

### Steps To Reproduce

Broswing to https://ip:3001

### Environment

```markdown
- OS: Ubuntu 20.04
- How docker service was installed: Compose
```

### CPU architecture

x86-64

### Docker creation

```bash
---
services:
  freetube:
    image: lscr.io/linuxserver/freetube:latest
    container_name: freetube
    security_opt:
      - seccomp:unconfined #optional
    environment:
      - PUID=1000
      - PGID=1000
      - TZ=Etc/UTC
    volumes:
      - /path/to/config:/config
    ports:
      - 3000:3000
      - 3001:3001
    shm_size: "1gb"
    restart: unless-stopped
```

### Container logs

```bash
[custom-init] No custom files found, skipping...

[ls.io-init] done.

Connection failure: Connection refused

pa_context_connect() failed: Connection refused

Connection failure: Connection refused

pa_context_connect() failed: Connection refused

_XSERVTransmkdir: ERROR: euid != 0,directory /tmp/.X11-unix will not be created.

Xvnc KasmVNC 1.3.2 - built Oct 26 2024 19:01:39

Copyright (C) 1999-2018 KasmVNC Team and many others (see README.me)

See http://kasmweb.com for information on KasmVNC.

Underlying X server release 12014000, The X.Org Foundation

Obt-Message: Xinerama extension is not present on the server

 2024-10-28 11:08:29,850 [INFO] websocket 0: got client connection from 127.0.0.1

 2024-10-28 11:08:29,870 [PRIO] Connections: accepted: @172.18.0.1_1730113709.850672::websocket

 2024-10-28 11:16:22,277 [PRIO] Connections: closed: @172.18.0.1_1730113709.850672::websocket (Clean disconnection)
```

**Comment by Vaslo** [comment]
> ### Is there an existing issue for this?
> * [x]  I have searched the existing issues
> 
> ### Current Behavior
> Installed Freetube using Docker Compose. Everything works fine apart from there's no sound.
> 
> ### Expected Behavior
> _No response_
> 
> ### Steps To Reproduce
> Broswing to https://ip:3001
> 
> ### Environment
> ```
> - OS: Ubuntu 20.04
> - How docker service was installed: Compose
> ```
> 
> ### CPU architecture
> x86-64
> 
> ### Docker creation
> ```shell
> ---
> services:
>   freetube:
>     image: lscr.io/linuxserver/freetube:latest
>     container_name: freetube
>     security_opt:
>       - seccomp:unconfined #optional
>     environment:
>       - PUID=1000
>       - PGID=1000
>       - TZ=Etc/UTC
>     volumes:
>       - /path/to/config:/config
>     ports:
>       - 3000:3000
>       - 3001:3001
>     shm_size: "1gb"
>     restart: unless-stopped
> ```
> 
> ### Container logs
> ```shell
> [custom-init] No custom files found, skipping...
> 
> [ls.io-init] done.
> 
> Connection failure: Connection refused
> 
> pa_context_connect() failed: Connection refused
> 
> Connection failure: Connection refused
> 
> pa_context_connect() failed: Connection refused
> 
> _XSERVTransmkdir: ERROR: euid != 0,directory /tmp/.X11-unix will not be created.
> 
> Xvnc KasmVNC 1.3.2 - built Oct 26 2024 19:01:39
> 
> Copyright (C) 1999-2018 KasmVNC Team and many others (see README.me)
> 
> See http://kasmweb.com for information on KasmVNC.
> 
> Underlying X server release 12014000, The X.Org Foundation
> 
> Obt-Message: Xinerama extension is not present on the server
> 
>  2024-10-28 11:08:29,850 [INFO] websocket 0: got client connection from 127.0.0.1
> 
>  2024-10-28 11:08:29,870 [PRIO] Connections: accepted: @172.18.0.1_1730113709.850672::websocket
> 
>  2024-10-28 11:16:22,277 [PRIO] Connections: closed: @172.18.0.1_1730113709.850672::websocket (Clean disconnection)
> ```

Also having the same issue, running from an Unraid container.  I checked sound on my machine and in other containers and those work fine.

**Comment by thelamer** [comment]
I cannot replicate this on my machines, I wonder if there is a specific host dep that is missing. 

Connection refused would indicate that pulse is running but in a weird state. 

In the container you can right click the background and open xterm, from there run `ps aux |grep pulse`, is the service running ?

**Comment by NicoG96** [comment]
> In the container you can right click the background and open xterm, from there run `ps aux |grep pulse`, is the service running ?

```log
# ps aux | grep pulse
root        45  0.0  0.0    220    76 ?        S    12:27   0:00 s6-supervise svc-pulseaudio
abc        191  0.3  0.0 1116504 10112 ?       Ssl  12:27   0:02 /usr/bin/pulseaudio --log-level=0 --log-target=stderr --exit-idle-time=-1
root      9492  0.0  0.0 177648  2196 pts/0    S+   12:39   0:00 grep pulse
```

**Comment by elmartinoAT** [comment]
I have allways no sound after opening FreeTube because kasmvnc audio is deactivated at start,
when i activate the 'loudspeaker-symbol' of kasmvnc (in the top middle overlay ... with folder and mic) everything works fine.

Also when i reload the page, sound is gone again (symbol deactivated again). but can allways be activated again.

latest, docker on synology, tested on Chrome,FF,Edge

Is there an option to start FreeTube/kasmvnc directly with sound activated and remember these setting?

**Comment by thelamer** [comment]
No, the image works like this for universal browser support. You need to interact with the page for me to play sound and in many browsers the activation of sound is tied to a button press.

**Comment by NicoG96** [comment]
Wow, didn't even see the kasm overlay. Clicking the audio button fixed my issue. Not the most streamlined experience but it works now at least. Thank you @elmartinoAT!

**Comment by elmartinoAT** [comment]
> No, the image works like this for universal browser support....

Well thats ok and very usefull but some options will not break this.

And that somebody dont see the overlay is fact, i recognized it also very late on searching for audio-support.

This overlay buttons has to go also to the settings-page of kasm or somewhere else (e.g. freetube-settings).

For the first dirty try i mapped the index.html of the kclient out off the container and changed it only a little bit to work without button press (point body-onload-event to the audio() js function so the toggle of audio runs on open the page).

By the way there are some failures in the kclient index.html ... loading a deleteted js-script ...
